src/xdocs/config_coding.xml

```diff
@@ -3468,6 +3468,22 @@ public class D {
 }
        </source>
        <p>
+         3) If validateOnlyOverlapping is set to false but the variable is locally defined, there
+            will not be a violation provided the variable doesn't overlap, for example:
```

Comment:
Done. Now everything is in the next paragraph. Renamed the class from 'E' to 'C'.